README.md

```diff
@@ -61,9 +62,9 @@ are found.
 
 ### `monitor`
 
-Takes a snapshot of your project's dependency tree and monitors it on [snyk.io](https://snyk.io).
-You'll be alerted when new relevant vulnerabilities, updates or patches are
-disclosed.
+Takes a snapshot of your project's dependency tree and monitors it
+on [snyk.io](https://snyk.io). You'll be alerted when new relevant
```

Comment:
I'd add a new line before the dot to make this more clear

Reply:
I think they belong in the same paragraph for context. But this PR hasn't changed this, we didn't run it through a formatter so it moved things around.